src/main/treeDataProviders/utils/dependencyUtils.ts

```diff
@@ -541,6 +540,18 @@ export class DependencyUtils {
         }
     }
 
+    private static getUsageFeaturesByExistTech(projectDescriptors: Map<PackageType, vscode.Uri[]>, scanSuffix: string): IUsageFeature[] {
+        let featureArray: IUsageFeature[] = [];
+        for (const [techEnum, descriptors] of projectDescriptors.entries()) {
+            // Only add to usage if found descriptors for tech.
+            if (!!descriptors) {
```

Comment:
The descriptors could be empty [] and this will return true.

Reply:
added a check to see if length > 0 as well